src/com/palmergames/bukkit/config/ConfigNodes.java

```diff
@@ -1752,8 +1752,178 @@
-			"# Only under affect when explosions_break_blocks is true.");
+			"# Only under affect when explosions_break_blocks is true."),
 
+	WAR_SIEGE(
+			"war.siege",
+			"",
+			"############################################################",
+			"# +------------------------------------------------------+ #",
+			"# |                   Siege-War settings                 | #",
+			"# |                                                      | #",
+			"# |              Separate from other war modes           | #",
+			"# |                                                      | #",
+			"# |  This war feature is new/being trialled and          | #",
+			"# |  at present not fully supported by core Towny team.  | #",
+		    "# |  In an emergency contact Goosius                     | #",
+			"# |  on the towny support discord channel                | #",
+			"# |  or email anytime to Goosius1@gmail.com              | #",
+			"# +------------------------------------------------------+ #",
+			"############################################################",
+			""),
+	//Switches
+	WAR_SIEGE_ENABLED(
+			"war.siege.switches.enabled",
+			"false",
+			"# If true, the Siege-War feature is enabled.",
+			"# if false, the Siege-War feature is disabled."),
+	WAR_SIEGE_ATTACK_ENABLED(
+			"war.siege.switches.attack_enabled",
+			"true",
+			"# If true, then nations can start sieges."),
+	WAR_SIEGE_ABANDON_ENABLED(
+			"war.siege.switches.abandon_enabled",
+			"true",
+			"# If true, then nations can abandon sieges."),
+	WAR_SIEGE_TOWN_SURRENDER_ENABLED(
+			"war.siege.switches.town_surrender_enabled",
+			"true",
+			"# If true, then a town can surrender if there is only 1 nation attacker."),
+	WAR_SIEGE_INVADE_ENABLED(
+			"war.siege.switches.invade_enabled",
+			"true",
+			"# If true, then a nation siege winner can invade the defeated town.",
+			"# This action will add the town to the nation"),
+	WAR_SIEGE_PLUNDER_ENABLED(
+			"war.siege.switches.plunder_enabled",
+			"true",
+			"# If true, then a nation siege winner can plunder the defeated town.",
+			"# This action will steal money from the town.",
+			"# If the town does not have sufficient funds, it will be ruined."),
+	WAR_SIEGE_TOWN_LEAVE_DISABLED(
+			"war.siege.switches.nation_leave_disabled",
+			"true",
+			"#. If true, then a town cannot leave a nation of its own accord. ",
+			"# However the nation can always kick."),
+	WAR_SIEGE_REVOLT_ENABLED(
+			"war.siege.switches.revolt_enabled",
+			"true",
+			"#. If true, then a town can 'revolt' against the nation and leave",
+			 "# Usually enabled in combination with WAR_SIEGE_TOWN_LEAVE_DISABLED"),
+	WAR_SIEGE_PVP_ALWAYS_ON_IN_BESIEGED_TOWNS(
+			"war.siege.switches.pvp_always_on_in_besieged_towns",
+			"true",
+			"# If true, then pvp is always set to on during sieges",
+			"# Pvp returns to its previous setting when the siege ends"),
+	WAR_SIEGE_SPAWNING_DISABLED_NEAR_SIEGES(

```

Comment:
🚧 
- I noticed a possible issue today with this non-essential config: "block spawning near sieges".
- The issue is that if spawning is blocked while a 'town spawn on death' config is enabled,
  the the player could possibly be prevented from spawning at all.
- I haven't confirmed the issue on my test server yet, will look into it later today.

Reply:
* Completely removed the 'prevent spawn during siege' feature, because:
1. It is non-essential for an initial release
2. The above issue could be complex to resolve
3. The pattern of "complete spawn prevention" does not really fit into the general siegewar vibe of a more geopolitical but less-destructive war mode. e.g. complete spawn prevention to a town for an extended time could be just as destructive to the town organisation/recruitment/commerce etc., as the destruction of buildings.
4. The original idea behind the spawn prevention was to ensure residents felt more highly involved in sieges, however this is technically already done by turning pvp on.
5. In future, a less extreme version of this restriction could be developed which would suit the mode, for example, by an added delay to spawn.